src/utils/index.test.ts

```diff
@@ -328,7 +328,7 @@ describe('Utils', () => {
         params: {
           val: null,
         },
-        output: `null`,
+        output: `undefined`,
```

Comment:
Shouldn't this still result in `null`, if `null` is the provided value?

I guess the question is: Do we treat both `null` and `undefined` as "not set", or only `undefined`?

Reply:
> We need to use null as a value, since we're using JSON. In that way we have the optional params, but we don't assign them.

I don't think I understand. Isn't `null` the value here (since `params.val` is `null`) and therefore the JSON output should also be `null`?

> Since null is falsy, it won't get replaced and undefined is returned.

How does this behave if for instance `false` is passed? Thinking that that is also a falsy value, but a valid one that should be used in JSON (rather than `undefined`).

Reply:
`false` as a value is valid, and the test passes for this input:

https://github.com/GoogleChromeLabs/third-party-capital/blob/a317a227ef68b5bcbcc9e3caadfe4aecdb68aa92/src/utils/index.test.ts#L318C7-L324C9